components/operators/openshift-pipelines/operator/overlays/ibmcloud/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+
+resources:
+  - ../../base
+patches:
+  - path: patch-channel.yaml
+    target:
+      group: operators.coreos.com
+      kind: Subscription
+      name: openshift-pipelines-operator
+      namespace: openshift-operators
+      version: v1alpha1
+  - patch: |-
```

Comment:
This is mostly personal preference, but I like to keep the target above the patch/path.

I feel like it makes it easier to read.